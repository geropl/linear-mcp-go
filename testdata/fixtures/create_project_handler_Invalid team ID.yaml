---
version: 2
interactions:
    - id: 0
      request:
        proto: HTTP/1.1
        proto_major: 1
        proto_minor: 1
        content_length: 357
        transfer_encoding: []
        trailer: {}
        host: api.linear.app
        remote_addr: ""
        request_uri: ""
        body: '{"query":"\n\t\tmutation ProjectCreate($input: ProjectCreateInput!) {\n\t\t\tprojectCreate(input: $input) {\n\t\t\t\tsuccess\n\t\t\t\tproject {\n\t\t\t\t\tid\n\t\t\t\t\tname\n\t\t\t\t\tdescription\n\t\t\t\t\tslugId\n\t\t\t\t\tstate\n\t\t\t\t\turl\n\t\t\t\t}\n\t\t\t}\n\t\t}\n\t","variables":{"input":{"name":"Test Project 3","teamIds":["invalid-team-id"]}}}'
        form: {}
        headers:
            Content-Type:
                - application/json
        url: https://api.linear.app/graphql
        method: POST
      response:
        proto: HTTP/2.0
        proto_major: 2
        proto_minor: 0
        transfer_encoding: []
        trailer: {}
        content_length: -1
        uncompressed: true
        body: |
            {"errors":[{"message":"Argument Validation Error","path":["projectCreate"],"locations":[{"line":3,"column":4}],"extensions":{"code":"INVALID_INPUT","validationErrors":[{"target":{"name":"Test Project 3","teamIds":["invalid-team-id"]},"value":["invalid-team-id"],"property":"teamIds","children":[],"constraints":{"isUuid":"each value in teamIds must be a UUID"}}],"type":"invalid input","userError":true,"userPresentableMessage":"each value in teamIds must be a UUID."}}],"data":null}
        headers:
            Alt-Svc:
                - h3=":443"; ma=86400
            Cache-Control:
                - no-store
            Cf-Cache-Status:
                - DYNAMIC
            Content-Type:
                - application/json; charset=utf-8
            Etag:
                - W/"1e4-JRWXeyaRzkdssN5PwkmdFPPwZ6Y"
            Server:
                - cloudflare
            Vary:
                - Accept-Encoding
            Via:
                - 1.1 google
        status: 200 OK
        code: 200
        duration: 0s
