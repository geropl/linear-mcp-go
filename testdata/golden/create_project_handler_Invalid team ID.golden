err: 'Failed to create project: GraphQL error: Argument Validation Error'
output: ""
